vault/tls/server.cert

```diff
@@ -0,0 +1,8 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
is it possible to auto generate this per dev? this may expire at any given point